lib/screens/home/explorer/explorer_bloc.dart

```diff
@@ -0,0 +1,68 @@
+import 'dart:async';
+
+import 'package:get_it/get_it.dart';
+import 'package:provenance_wallet/services/models/provenance_validator.dart';
+import 'package:provenance_wallet/util/strings.dart';
+
+class ExplorerBloc extends Disposable {
+  @override
+  FutureOr onDispose() {
+    // TODO: implement onDispose
+    throw UnimplementedError();
+  }
+}
+
+class StakingDetails {
+  StakingDetails({
+    required this.delegates,
+    required this.validators,
+    this.selectedType = Strings.dropDownDelegate,
+    this.selectedStatus = Strings.dropDownAllStatuses,
+    required this.address,
+  });
+
+// FIXME: delegates are a different type eventually
+  List<ProvenanceValidator> delegates;
+  List<ProvenanceValidator> validators;
+  String selectedType;
+  String selectedStatus;
+  String address;
+}
+
+enum ValidatorType {
+  delegations,
+  redelegations,
+  unbonding,
+}
+
+enum ValidatorStatus {
+  active,
+  candidate,
+  jailed,
+}
+
+extension ValidatorStatusStuff on ValidatorStatus {
+  String get dropDownTitle {
+    switch (this) {
+      case ValidatorStatus.active:
+        return Strings.dropDownActive;
+      case ValidatorStatus.candidate:
+        return Strings.dropDownCandidate;
+      case ValidatorStatus.jailed:
+        return Strings.dropDownJailed;
+    }
+  }
+}
+
+extension ValidatorTypeStuff on ValidatorType {
```

Comment:
Opinion, but I think it is nice to name extensions `<Type>Extension` as a convention. `Stuff` is perhaps a bit too vague.

Reply:
Agreed.

Reply:
I did this as a temporary thing because I had ideas I just wanted to try and it worked so I forgot to rename.